src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
+    private int number;
```

Comment:
Similarly, a getter for the 'number' field should be provided for the same reasons as the 'colour' field.